content/ja/handbook/company/working-groups/customer-use-case-adoption.md

```
@@ -0,0 +1,100 @@
+2023-08-17 より、顧客ユースケース採用ワーキンググループの焦点を明示的に CI 採用へと転換し、FY24-Q4 終了までに CI 採用 ARR を 50% に達成するために必要なクロスファンクショナルなワークストリームの推進に注力しました。合意されたワークストリームの観点から、FY24 の残り期間においてこのワーキンググループが注力したユースケースは CI のみとなりました。
+
+## 属性
+
+| プロパティ                     | 値         |
+|--------------------------------|------------|
+| 作成日                         | 2023-05-16 |
+| 目標終了日                     | 2025-01-31 |
+| Slack                          | [#wg_customer-use-case-adoption](https://gitlab.slack.com/archives/C0584NEKSRJ) |
+| Google Doc                     | [Customer Use Case Adoption Scrum - Agenda](https://docs.google.com/document/d/1WtwXCK1r7hoco5O8oW5SIKiIWtXDr_WOLeWcIaDM7Nk/edit?usp=sharing)  |
+| エピック - プライマリ           | [Use Case Adoption Measurement & Improvement](https://gitlab.com/groups/gitlab-com/-/epics/2190) |
+| 概要 & ステータス              | 以下の[終了基準](#exit-criteria)を参照 |
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)